modules/imgur.py

```diff
@@ -18,6 +22,95 @@ def _prefix(self, data):
             text += "%s " % data["account_url"]
         return text
 
+    @utils.hook("received.message.channel",
+                priority=EventManager.PRIORITY_LOW)
+    def channel_message(self, event):
+        if not event["channel"].get_setting("auto-imgur", False):
+            return
+
+        self._imgur(event)
+
+    def _imgur(self, event):
```

Comment:
why is this it's own function? only 1 thing references it

Reply:
Fixed.